code/go/pkg/validator/validator_test.go

```diff
@@ -813,6 +813,12 @@ func TestValidateForbiddenDataStreamName(t *testing.T) {
 	}
 }
 
+func TestSharedFolderAndIgnoreLinks(t *testing.T) {
+	pkgRootPath := filepath.Join("..", "..", "..", "..", "test", "packages", "with_links")
+	errs := ValidateFromPath(pkgRootPath)
+	require.NoError(t, errs)
```

Comment:
Nit. No special handling is done for this package, I think it could be directly added to [TestValidateFile](https://github.com/elastic/package-spec/blob/7605ac8d476aba086fa791ef16c94b53b26ea19f/code/go/pkg/validator/validator_test.go#L24).